src/pool/srv_target.c

```diff
@@ -257,13 +257,18 @@ pool_child_delete_one(void *uuid)
 	ds_pool_child_put(child); /* -1 for the list */
 
 	ds_pool_child_put(child); /* -1 for lookup */
+
+	/*
+	 * FIXME: Need to wait for last reference of ds_pool_child dropped,
+	 * since the ds_pool_child references ds_pool by 'spc_pool' without
+	 * holding ds_pool refcount.
+	 */
```

Comment:
Let's fix this in later follow-on patches.

The major use cases of 'spc_pool' is from DTX & EC aggregation (accessing ds_pool->sp_map), now we can guarantee all aggregation ULTs have already exited here. @Nasf-Fan what about DTX?